starter-code/functions-and-arrays.js

```diff
@@ -79,6 +120,21 @@ var words = [
   "simple",
   "bring"
 ];
+
```

Comment:
Chicos pegadle un ojo a indexOf() cualquier duda ya sabéis: a MDN o W3S... o nosotros xD